app/components/Button.tsx

```diff
@@ -8,6 +8,8 @@ interface ButtonProps extends React.PropsWithChildren<Props> {
   fill?: 'fill' | 'outline' | 'flat'
   label?: string
   margin?: string
+  delayLongPress?: number
+  onLongPress?: () => void
```

Comment:
This prop should be refactored to extends`React.PropsWithChildren<TouchableOpacityProps>`, to open an issue to resolve in another PR

Reply:
@thedoublejay open an issue for this